Index: b/x/3/1/5/trunk/suite.rc
===================================================================
--- b/x/3/1/5/trunk/suite.rc	(revision 174521)
+++ b/x/3/1/5/trunk/suite.rc	(revision 178901)
@@ -83,5 +83,8 @@

         [[[ {{EXPT_RESUB}} ]]]
-            graph = atmos_main => {{ RESUB_GRAPH }}
+            graph = """
+                    atmos_main => {{ RESUB_GRAPH }}
+                    postproc[+{{EXPT_RESUB}}] => tracking
+            """

         {% if TASK_ARCH_WALL %}
@@ -204,4 +207,16 @@
             ROSE_TASK_APP = fcm_make_pp

+    [[tracking]]
+        inherit = RUN_MAIN, TRACKING_RESOURCE
+        pre-script = """
+            module load scitools
+            module load um_tools
+            module load gcc/7.3.0
+            module load cray-netcdf/4.3.2"""
+        [[[environment]]]
+            ROSE_TASK_APP = tempest_tracker
+            TIME_CYCLE = $({{ROSEDATE}} ref)
+            PREVIOUS_CYCLE = $({{ROSEDATE}} --offset=-{{EXPT_RESUB}} ref)
+
     [[pptransfer]]
         inherit = RUN_MAIN, PPTRANSFER_RESOURCE